
# REQUEST
#===============================================================================
Header header

# 2D points representing two diagonally opposite corners of the 2D region of interest
# (the first element of the array = X-coord, the second = Y-coord)
int16[2] p1
int16[2] p2

# Estimation mode {1, 2, 3} - if it is not specified, the default value = 1
int8 mode
---

# RESPONSE
#===============================================================================

# pose includes a position and an orientation of the resulting bounding box
# reference point is the center of bounding box base
geometry_msgs/Pose pose

# size of the bounding box in direction of X (length), Y (width) and Z (height) coordinates 
# (before rotation given by orientation, which is included in pose)
# l = 0.5 * full_length
# w = 0.5 * full_width
# h = full_height
geometry_msgs/Point bounding_box_lwh

